versions.sh

```diff
@@ -83,6 +83,7 @@ for version in "${versions[@]}"; do
 	doc="$(jq <<<"$doc" -c '
 		.variants = [
 			(
+				"trixie",
 				"bookworm",
 				"bullseye",
```

Comment:
This PR should also drop `bullseye` (and all references to it).

Reply:
Done